app/scripts/metamask-controller.js

```diff
@@ -81,7 +84,7 @@ module.exports = class MetamaskController {
     const noticeController = this.noticeController
 
     return {
-      getState: (cb) => { cb(null, this.getState()) },
+      getState: nodeify(this.getState.bind(this)),
```

Comment:
maybe we can rename `nodeify` to something more semantic


`fromPromise`
`toCallback`
`promise2cb` 
`promiseToCb`
`await` heh
idk